egs/wsj/s5/steps/cleanup/internal/resolve_ctm_edits_overlaps.py

```diff
@@ -136,8 +136,8 @@ def wer(ctm_edit_lines):
         return float('inf')
     if num_words == 0 and num_incorrect_words == 0:
         return 0
-    return (float(num_incorrect_words) / num_words, -num_words)
-
+    #return (float(num_incorrect_words) / num_words, -num_words)
+    return (float(num_incorrect_words) / num_words)
```

Comment:
Perhaps @vimalmanohar  could comment.

Reply:
I think @jtrmal is also working on fixing the exact same issue.

Reply:
hm, so the proper fix isn't as simple as this?